app/src/main/res/values-ne/strings.xml

```diff
@@ -1047,7 +1047,7 @@
     <string name="below_min_apk">एप संस्करण अनुमत संस्करणभन्दा कम छ। कृपया अनुप्रयोगलाई नवीनतम संस्करणमा अद्यावधिक गर्नुहोस्।</string>
     <string name="add_note">नोट थप्नुहोस् (वैकल्पिक)</string>
     <string name="visit_count">%s (%d %s)</string>
-    <string name="no_teams">टोलीहरू उपलब्ध छैनन्</stri
+    <string name="no_teams">टोलीहरू उपलब्ध छैनन्</string>
```

Comment:
this is causing builds to fail and for some reason is in the master branch, so i think this change should be pushed to master asap